packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -200,86 +228,193 @@ export class VaMemorableDate {
       handleDateChange,
       handleDateKey,
       value,
+      uswds,
+      monthSelect,
     } = this;
 
     const [year, month, day] = (value || '').split('-');
     const describedbyIds = ['dateHint', hint ? 'hint' : '']
       .filter(Boolean)
       .join(' ');
 
+    this.populateOptions();
```

Comment:
I'll look into it

Reply:
I tried using `componentDidLoad` but when one change the language, the re-render is not changing the options anymore. I think it will need to run in every re-render. Any thoughts?

Reply:
Strangely, I can't get the options to translate at all (even in its current location). Was there a regression that made it stop working maybe? 

<img width="350" alt="Screenshot 2023-03-06 at 3 28 45 PM" src="https://user-images.githubusercontent.com/872479/223234995-42b6b358-e346-44f4-82b9-710fdea2498a.png">

Reply:
Ah, I see. So it still only works with the onChange. My feeling here is that if it's not fully working, we probably shouldn't add partial support and we should instead just tackle adding the full support in the separate ticket you're going to write up (removing any partial support from this PR). This is probably also affecting `va-date` as well. 

But what are your feelings about it @Andrew565?